package.json

```diff
@@ -84,10 +84,10 @@
     "demo:min": "cross-env RXP_METAPLAYLIST=true RXP_MINIFY=true node ./scripts/generate_full_demo.js",
     "demo:watch": "cross-env RXP_METAPLAYLIST=true node ./scripts/generate_full_demo.js --watch",
     "doc": "rm -rf doc/generated; node ./scripts/doc-generator/index.js doc/ doc/generated",
-    "start:docs": "docusaurus start docs/",
-    "build:docs": "docusaurus build docs/",
-    "deploy": "docusaurus deploy docs/",
-    "serve": "docusaurus serve docs/",
+    "start:docs": "docusaurus start docusaurus/",
```

Comment:
Why the "docusaurus" root name?

Not a fan of naming one of our directory to the used dependency (I know the project is called RxPlayer but still :p). `doc`/`docs` is much more understandable IMO.

Is it because if you did so you are forced to create a `/docs/docs` directory or similar?